src/api/index.tsx

```diff
@@ -542,3 +543,15 @@ export const deleteComment = async ({
     method: HttpMethod.DELETE,
   });
 };
+
+type GetValidationParams = {
+  apiState: ApiState;
+  url: string;
+};
+
+export const getValidation = async ({ apiState, url }: GetValidationParams) => {
+  return callApi<ResponseOnly<ExternalLinterResult>>({
+    apiState,
+    endpointUrl: url,
```

Comment:
Can we construct the endpoint here instead? We don't need to pass the `url` anymore I think. We can probably pass the addon ID.

Reply:
We need the file ID as well, but yes, we should be able to do this. I'll file a follow-up.